src/GraphQL/DocumentElementType/BlockType.php

```diff
@@ -35,15 +35,15 @@ public static function getInstance()
                         '_editableName' => [
                             'type' => Type::string(),
                             'resolve' => static function ($value = null, $args = [], $context = [], ResolveInfo $resolveInfo = null) {
-                                if ($value instanceof Checkbox) {
```

Comment:
shouldn't it be 
$value instance of Pimcore\Model\Document\Editable\Block instead ?

Besides that, since this is a bug ... could you please rebase to 1.0 ?